### DISCUSSION TITLE
Top Oferta po API

### DISCUSSION THREAD

**Discussion by tomasz-grzesik** [allegro/allegro-api#9370] [Suggestion - verification]
Wraz z przechodzeniem z widoków ofertowych na produktowe warto pomyśleć o dodaniu informacji o Top Ofercie po API - osobny endpoint, który po podaniu id produktu zwróci chociażby id oferty, aby sprzedawca mógł wiedzieć, że jego oferta nie jest topowa i mógł zachęcić się np. do włączenia wyróżnienia, czy obniżenia ceny.

**Comment by tomasz-grzesik** [comment]
Podbijam temat. Minęło kilka lat, a tej informacji ciągle brak w danych ofert sprzedawcy.

**Comment by MaciejFrackowiak** [comment]
@tomasz-grzesik Na ten moment nic się nie zmieniło - nie ma decyzji by wprowadzić tu zmiany.
